### ISSUE TITLE
[Designer] [Home - Adaptive Cards - New Card]: ‘Blank Card’ is not accessible with keyboard. 

### ISSUE THREAD

**Issue by JeanRoca** [microsoft/AdaptiveCards#7687]
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/2524423
